python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/mpls.schema.yml

```diff
@@ -33,3 +33,256 @@ keys:
+            type: str
+            valid_values:
+              - implicit-null
+              - explicit-null
+          preemption_method:
+            type: dict
+            keys:
+              preemption:
+                type: str
+                required: true
+                valid_values:
+                  - hard
+                  - soft
+              timer:
+                description: Timer value in units of seconds.
+                type: int
+                convert_types:
+                  - str
+                min: 1
+                max: 65535
+          mtu_signaling:
+            description: Enable MTU signaling.
+            type: bool
+          graceful_restart:
+            description: RSVP graceful restart.
+            type: dict
+            keys:
+              role_helper:
+                type: dict
+                keys:
+                  enabled:
+                    type: bool
+                    required: true
+                  timer_recovery:
+                    type: int
+                    convert_types:
+                      - str
+                    min: 1
+                    max: 320
+                    description: Maximum recovery timer value in seconds.
+                  timer_restart:
+                    type: int
+                    convert_types:
+                      - str
+                    min: 1
+                    max: 320
+                    description: Maximum restart timer value in seconds.
+              role_speaker:
+                type: dict
+                keys:
+                  enabled:
+                    type: bool
+                    required: true
+                  timer_recovery:
+                    type: int
+                    convert_types:
+                      - str
+                    min: 1
+                    max: 320
+                    description: Maximum recovery timer value in seconds.
+                  timer_restart:
+                    type: int
+                    convert_types:
+                      - str
+                    min: 1
+                    max: 320
+                    description: Maximum restart timer value in seconds.
+          hello:
+            type: dict
+            keys:
+              interval:
+                description: Time between hello messages in seconds.
+                type: int
+                required: true
+                convert_types:
+                  - str
+                min: 1
+                max: 60
+              multiplier:
+                description: Number of missed hellos after which the neighbor is expired.
+                type: int
+                required: true
+                convert_types:
+                  - str
+                min: 1
+                max: 255
+          hitless_restart:
+            description: RSVP hitless restart.
+            type: dict
+            keys:
+              enabled:
+                type: bool
```

Comment:
we can make enabled as required key

Reply:
As Claus suggested we need to avoid setting required keys, I have removed all but one(authentication.password_indexes.password) required keys from schema.